flow360/component/simulation/operating_condition.py

```diff
@@ -4,47 +4,24 @@
 
 from flow360.component.simulation.framework.base_model import Flow360BaseModel
 
-"""
-    Defines all the operating conditions for different physics.
-    The type of operating condition has to match its volume type.
-    Operating conditions defines:
-    1. The physical (non-geometrical) reference values for the problem.
-    2. The initial condition for the problem.
-
-    TODO:
-    1. What other types of operation conditions do we need?
-"""
-
-
-class TurbulenceQuantities(Flow360BaseModel):
-    """PLACE HOLDER, Should be exactly the the same as `TurbulenceQuantitiesType` in current Flow360Params"""
-
-    pass
 
+class DensityTemperature(Flow360BaseModel):
+    temperature: TemperatureType = pd.Field(288.15)
+    density: DensityType = pd.Field(1.225)
 
-class ExternalFlowOperatingConditions(Flow360BaseModel):
-    Mach: float = pd.Field()
-    alpha: float = pd.Field(0)
-    beta: float = pd.Field(0)
-    temperature: float = pd.Field(288.15)
-    reference_velocity: Optional[float] = pd.Field()  # See U_{ref} definition in our documentation
 
-    initial_flow_condition: Optional[tuple[str, str, str, str, str]] = pd.Field(
-        ("NotImplemented", "NotImplemented", "NotImplemented")
-    )
-    turbulence_quantities = Optional[TurbulenceQuantities] = pd.Field()
+class StandardAtmosphere(Flow360BaseModel):
+    altitude: LengthType = pd.Field(0)
+    temperature_offset: TemperatureType = pd.Field(0)
 
 
-class InternalFlowOperatingConditions(Flow360BaseModel):
-    pressure_difference: float = pd.Field()
-    reference_velocity: float = pd.Field()
-    inlet_velocity: float = pd.Field()
-
-
-class SolidOperatingConditions(Flow360BaseModel):
-    initial_temperature: float = pd.Field()
-
+class OperatingConditions(Flow360BaseModel):
+    """
+    Defines all the operating conditions for different physics.
+    The type of operating condition has to match its volume type.
```

Comment:
Do we still plan to allow operating condition to be overwritten per volume?

Reply:
I think we decided not to allow per volume operating condition right? If the user wants a different freestream they can just change the boundary condition, so there's no use of operation condition per zone